config/tools-manual.json

```diff
@@ -1,23 +1,267 @@
+                "filters": {
+                    "language": "JavaScript",
+                    "technology": ["Node js", "Liquid"],
+                    "categories": ["api", "directory"]
+                }
+            },
+            {
+                "title": "API Tracker - AsyncAPI specs",
+                "description": "Explore APIs and companies with public AsyncAPI specifications.",
+                "links": {
+                    "websiteUrl": "https://apitracker.io/specifications/asyncapi"
+                },
+                "filters": {
+                    "categories": ["api", "directory"]
+                }
+            },
+            {
+                "title": "Check-API",
+                "description": "Check_api is a simple utility wrapping a collection of API definition format validators. It allows you to validate a local file or remote URL with a single command-line or programmatic invocation.",
+                "links": {
+                    "repoUrl": "https://github.com/Mermade/check_api"
+                },
+                "filters": {
+                    "language": "JavaScript",
+                    "technology": ["Node js"],
+                    "categories": ["api","validator"]
+                }
+            }
+        ]
     },
     "Code-first tools": {
         "description": "The following is a list of tools that generate AsyncAPI documents from your code.",
-        "toolsList": []
+        "toolsList": [
+            {
+                "title": "Go AsyncAPI",
+                "description": "This library helps to create AsyncAPI spec from your Go message structures. It uses reflection to translate Go structures in JSON Schema definitions and arrange them in AsyncAPI schema.",
+                "links": {
+                    "repoUrl": "https://github.com/swaggest/go-asyncapi"
+                },
+                "filters": {
+                    "language": "Go",
+                    "categories": ["code-first"]
+                }
+            },
+            {
+                "title": "Saunter",
+                "description": "Saunter is an AsyncAPI documentation generator for dotnet. Generates (and hosts) an AsyncAPI schema document from your code.",
+                "links": {
+                    "repoUrl": "https://github.com/tehmantra/saunter"
+                },
+                "filters": {
+                    "language": "C#",
+                    "technology": [".NET", "ASP.NET"],
+                    "categories": ["code-first"]
+                }
+            },
+            {
+                "title": "Java AsyncAPI",
+                "description": "This tool stores modules, which simplifies interacting with AsyncAPI in jvm ecosystem.",
+                "links": {
+                    "repoUrl": "https://github.com/Pakisan/jasyncapi"
+                },
+                "filters": {
+                    "language": "Java",
+                    "technology": ["Kotlin", "Maven"],
```

Comment:
don't want to be a party pooper, but `Kotlin` is a programming language and looks like we have a case where we can have 2 languages. Like in general in case of JS/TS.

technically it should not be that hard to support. We can support `languages` as both, string and array, so there is no need to update existing `.asyncapi-tool` files 🤔

Reply:
So should I change the type of `language` to both string and array of string in schema? It can take some major changes because in this case, we need to change both backend functions and frontend implementation, including Filters and Contexts. 

I added Kotlin as same logic you provided for `TypeScript` once, so I thought that `Kotlin` can be added in the same way. Also, you can add any dynamic/custom technology you want, there is no issue regarding the matching of technologies with a defined list of technologies.

Reply:
I think it can be easily handled as a follow-up task. I'm also not 100% sure. I'm always confused what people want in case of JS. 

- If tool is written in JS, it can be used in any library in JS or TS
- If tool is written in TS, it can be used in any library in JS or TS

I have no idea if developers simply filter by default both JS and TS or not 😄 

So yeah, not critical, can be tackled in a separate issue

Reply:
Yeah, let's create an issue regarding this and have a chat with other contributors too to have an opinion on whether a language should be a string or an array of strings. Many of them will have different notions and ideologies. Let's have a discussion regarding this on a issue :sweat_smile: